### PR TITLE
tools: update binlog slave to binlog consumer

### PR DESCRIPTION
**Pull request by WangXiangUSTC**
### What is changed, added or deleted? (Required)

tools: update binlog slave to binlog consumer, should merge after https://github.com/pingcap/tidb-tools/pull/380/files

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [x] v3.1 (TiDB 3.1 versions)
- [x] v3.0 (TiDB 3.0 versions)
- [x] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [x] Change aliases
- [ ] Have version specific changes 
- [ ] Might cause conflicts

### CODE REVIEW

**Comment by WangXiangUSTC** [comment]
@ran-huang @july2993 PTAL

File: tidb-binlog/binlog-consumer-client.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
 ---
-title: Binlog Slave Client User Guide
-summary: Use Binlog Slave Client to consume TiDB slave binlog data from Kafka and output the data in a specific format.
+title: Binlog Consumer Client User Guide
+summary: Use Binlog Consumer Client to consume TiDB secondary binlog data from Kafka and output the data in a specific format.
 aliases: ['/docs/dev/tidb-binlog/binlog-slave-client/','/docs/dev/reference/tidb-binlog/binlog-slave-client/']
```
Comment (ran-huang) [inline tidb-binlog/binlog-consumer-client.md:4]:
```suggestion
aliases: ['/tidb/dev/binlog-slave-client','/docs/dev/tidb-binlog/binlog-slave-client/','/docs/dev/reference/tidb-binlog/binlog-slave-client/']
```

**Comment by ran-huang** [comment]
@WangXiangUSTC Made a few updates https://github.com/pingcap/docs/pull/3522/commits/4cbf07ea8e809624f9045115fb0eea1a989860e2. PTAL

**Comment by WangXiangUSTC** [comment]
> @WangXiangUSTC Made a few updates [4cbf07e](https://github.com/pingcap/docs/commit/4cbf07ea8e809624f9045115fb0eea1a989860e2). PTAL

ok

**Pull request merged**